org.eclipse.gef.dot/src/org/eclipse/gef/dot/internal/language/parser/antlr/lexer/Lexer.java

```diff
@@ -26,8 +26,7 @@
 /**
  * This derived class includes the fix
  * https://github.com/eclipse/xtext-core/commit/34d61d4d80c6992045ed013a0ac4eb337a1f4e87
- * available from Xtext Version 2.9 (MARS). TODO: drop this class as soon as
- * MARS will be the minimum supported platform.
+ * available from Xtext Version 2.9 (MARS).
```

Comment:
Is there a reason for not removing this class?

Reply:
This class can be removed after the merge. However, first we should wait for that the GEF Jenkins Jobs are running again (`Bugzilla 559286`) to see which Xtext version will be the minimum required by GEF DOT.